lib/engine/game/g_1832/game.rb

```diff
@@ -242,6 +248,43 @@ def setup
           @tile_144 ||= @all_tiles.find { |t| t.name == '144' }
         end
 
+        def available_programmed_actions
+          [Action::ProgramMergerPass, Action::ProgramBuyShares, Action::ProgramSharePass]
+        end
+
+        def merge_rounds
+          [G1832::Round::Merger]
+        end
+
+        def corps_available_for_systems
+          @corporations.select { |c| c.operated? && c.type != :system }
+        end
+
+        # def next_round!
+        #   @round =
+        #     case @round
+        #     when G1870::Round::Stock
+        #       if merge_corporations.any? && phase.name.to_i == 4 || 5
+        #         @log << "-- #{round_description('Merger', @round.round_num)} --"
+        #         # The order of steps in the Grand Trunk Games rules is incorrect
+        #         # (confirmed by Ian D Wilson https://github.com/tobymao/18xx/issues/9655).
+        #         # It has buying shares before removing tokens.
+        #         G1832::Round::Merger.new(self, [
+        #           G1867::Step::PostMergerShares, # Step C & D
+        #           Engine::Step::DiscardTrain, # Step F
+        #           G1867::Step::Merge,
+        #         ], round_num: @round.round_num)
+        #       else
+        #         @operating_rounds = @phase.operating_rounds
+        #         reorder_players
+        #         new_operating_round
+        #       end
+        #     when G1832::Round::Merger
+        #       new_operating_round
+        #     when Engine::Round::Operating
+
+        # end
+
```

Comment:
Please remove this block of commented out code. Having dead code committed would make the git history messier than it needs to be.

Reply:
fair! I have a habit of leaving stuff I'm working on like that, but yeah it doesn't help anything for legibility.